src/Sylius/Bundle/ProductBundle/Doctrine/Query/Function/AbstractJsonFunctionNode.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
```

Comment:
Should it have some license block? If none will be added, then next coding style check will change it to sylius one.